whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/SponsoringOrgSpec.groovy

```diff
@@ -1187,7 +1187,51 @@ class SponsoringOrgSpec extends BaseQueryUpdateSpec {
         status | extra
         "ALLOCATED-BY-LIR"  | ""
         "AGGREGATED-BY-LIR" | "\nassignment-size: 96"
-        "ASSIGNED"          | ""
+    }
+
+
+    def "create inet6num with assigned status, with sponsoring org, with override"() {
```

Comment:
Do we need a test combining sponsoring-org and mnt-lower ? I think testing each functionality separately is enough

Reply:
I suggest removing this test, we don't need to test "sponsoring-org" and "mnt-lower" together